app.js

```diff
@@ -10,86 +11,116 @@ const result = document.querySelector(".result");
 const priceShoes = document.querySelector(".price");
 const carosel = document.querySelectorAll(".carosel img");
 
-const addToCartButton = document.querySelector('.add_toCart');
-const productInBasket = document.querySelector('.product_added_hide');
-const emptyBasketText = document.querySelector('.cart_empty_text');
+const addToCartButton = document.querySelector(".add_toCart");
+const productInBasket = document.querySelector(".product_added_hide");
+const emptyBasketText = document.querySelector(".cart_empty_text");
 
-const totalPriceBasket = document.querySelector('.total_price');
-const quantity = document.querySelector('.quantity');
-const iconDelete = document.querySelector('.icon_delete');
+const totalPriceBasket = document.querySelector(".total_price");
+const quantity = document.querySelector(".quantity");
+const iconDelete = document.querySelector(".icon_delete");
 
 const imagesCount = carosel.length;
-console.log(imagesCount);
-console.log(carosel[imagesCount - 1]);
+
+const desktopImagesThumbnail = document.querySelectorAll(".little_images img");
+const desktopCarousel = document.querySelector(".desktopCarousel_hide");
+const desktopDefaultProductImage = document.querySelector(
+  ".product_image_default_desktop"
+);
+const grayedOutBodyDesktop = document.querySelector(".grayedOut_body");
+const closeButtonDesktop = document.querySelector(".closeButton");
+const previousButtonDesktop = document.querySelector(".previousButtonDesktop");
+const nextButtonDesktop = document.querySelector(".nextButtonDesktop");
+
+const cartImage = document.querySelector(".cart");
+const cartCounter = document.querySelector(".itemsCountOnCart_hide");
+const cartCounterValue = document.querySelector(".cart_counter");
 
 let count = 1;
 let price = 125;
+const localStorageCount = localStorage.getItem("itemsInCart");
+console.log(localStorageCount);
 
-//Mobile Carousel
-//button next
-buttonNext.addEventListener("click", () => {
-  const currentImg = document.querySelector(".product_image_default");
-  const nextImg = currentImg.nextElementSibling;
+if (localStorageCount) {
+  productInBasket.classList.add("product_added");
+  emptyBasketText.style.display = "none";
+  cartCounter.classList.add("itemsCountOnCart");
 
+  result.innerHTML = localStorageCount;
+  quantity.innerHTML = localStorageCount;
+  cartCounterValue.innerHTML = localStorageCount;
+}
+
+//Mobile Carousel
```

Comment:
this comment is wrong, also always leave a space after // when commenting 

`// This is a comment.`